src/pages/Signup/index.js

```diff
@@ -0,0 +1,48 @@
+import './index.css';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { SignupFormComponent } from '@components/SignupForm';
+import { authAction } from '../../store/modules/auth/actions';
+import logo from '../../assets/images/logo.png';
+import SocialMediaLogin from '../../components/SocialMedia/SocialMediaLogin';
+
+export const SignupForm = ({ status, signUp, history, location }) => (
+  <div className="w-11/12 md:w-9/12 lg:w-7/12 mx-auto m-16 rounded-lg flex items-stretch bg-red-400 h-100">
+    <div className="md:flex max-w-lg w-7/12 bg-color height items-center signup-card hidden rounded-l-lg ">
+      <div className=" pl-8 font-bold text-left text-white text-2xl">
+        <div>Welcome back,</div>
```

Comment:
Change `Welcome back,` to `Welcome to ErrorSwag`.

Wrap this text in an `h1` tag or the most appropriate heading tag. This is important for accessibility and SEO.

Use `<br>` tags to break the text into multiple lines if needed.